### PR TITLE
Rename view to reshape

### PR DESCRIPTION
**Pull request by kevinstephano**
Renaming view as reshape since nvFuser's implementation is considered reshape to remove confusion.

### CODE REVIEW

File: third_party/nvfuser/csrc/lower_expr_sort.cpp [hunk 1]
```diff
@@ -1189,13 +1189,30 @@ bool ExprSegmentationSorter::supportedMerge(ExprGroup* sg1, ExprGroup* sg2) {
     return false;
```
Comment (csarofeen) [inline third_party/nvfuser/csrc/lower_expr_sort.cpp:1189]: Can you pull these changes out?

File: third_party/nvfuser/test/test_gpu3.cpp [hunk 2]
```diff
@@ -7352,6 +7351,55 @@ TEST_F(
       "");
 }
 
+TEST_F(NVFuserTest, FusionExprSortMatmulLikeSchedule_CUDA) {
```
Comment (csarofeen) [inline third_party/nvfuser/test/test_gpu3.cpp]: Can you pull this change out?

**Review by csarofeen — COMMENTED** [review]
LGTM just need to see the expression sorting changes removed.

File: third_party/nvfuser/csrc/lower_expr_sort.cpp (hunk 1 above)
Comment (kevinstephano) [reply third_party/nvfuser/csrc/lower_expr_sort.cpp:1189]: I reverted an incorrect merge.  Thanks for catching this!

File: third_party/nvfuser/test/test_gpu3.cpp (hunk 2 above)
Comment (kevinstephano) [reply third_party/nvfuser/test/test_gpu3.cpp]: Same incorrect merge.

**Comment by kevinstephano** [comment]
Closing as I seemed to mess up the merge in favor of new PR #2392.

**Pull request closed without merging**